### ISSUE TITLE
Event template doesn't match event

### ISSUE THREAD

**Issue by kailsar** [aws-samples/amazon-ecr-cross-region-replication#1]
I deployed this and it was failing to replicate - after some investigation I found that the event wasn't matching the template - perhaps the JSON has changed? This is what worked for me:

```     EventPattern: 
        source: ["aws.ecr"]
        detail:
          action-type: ["PUSH"]

**Comment by kailsar** [comment]
Also had to change the mapping like so:

```            InputPathsMap: 
              awsRegion: $.region
              repoName: $.detail.repository-name
              registryId: $.account
              imageTag: $.detail.image-tag

**Comment by walkley** [comment]
Thanks for your testing. I deployed the CFN and tested again, it works after I pushed imaged into ap-southeast-1 region.

This indeed reminder me to update the legacy CloudTrail API event in current implementation, though. As you mentioned above, ECR has image push/scan/delete events publish to EventBridge, which would be more clear to use it in this solution:

- https://aws.amazon.com/about-aws/whats-new/2019/11/ecr-events-now-published-to-eventbridge/?nc1=h_ls

- https://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr-eventbridge.html

I will update the CFN template with ECR events soon.
